style.css

```diff
@@ -32,13 +30,39 @@ body {
   margin: 0 auto;
 }
 
+.text-header {
+  text-align: center;
+}
+
 .first-section {
+  text-align: center;
+  padding: 50px 50px;
+}
+
+.title-first-section {
+  background-color: red;
+  color: #FFFFFF;
+  display: inline-block;
+  padding: 0 8px 0 8px;
+  font-weight: bold;
+}
+
+.first-text {
   background-color: #F4F4F4;
-  border: 4px solid red;
+  padding: 50px 50px;
+}
+
+.title-second-section {
+  background-color: red;
+  color: #FFFFFF;
+  display: inline-block;
+  padding: 2px 8px 2px 8px;
```

Comment:
You can use the shorthand for it. padding: 2px 8px;  the first one is for top and bottom and the second is for right and left. when the value for top and bottom and also left and right are the same, you should use it in that way.

Reply:
Done.